_episodes_rmd/13-dplyr.Rmd

```diff
@@ -263,6 +263,35 @@ gdp_pop_bycontinents_byyear <- gapminder %>%
               sd_gdp_billion=sd(gdp_billion))
 ```
 
+### Connect mutate with logical filtering: ifelse
+
+When creating new variables, we can hook this with a logical condition. A simple combination of 
+`mutate()` and `ifelse()` facilitates filtering right where it is needed: in the moment of creating something new.
+This easy-to-read statement is a fast and powerful way of discarding certain data (even though the overall dimension
+of the data.frame will not change) or for updating values depending on this given conditon.
+
+```{r}
+## keeping all data but "filtering" after a certain condition
+# calculate GDP only for people with a life expectation above 25
+gdp_pop_bycontinents_byyear_above25 <- gapminder %>%
+    mutate(gdp_billion= ifelse(lifeExp > 25, gdpPercap*pop/10^9, NA)) %>%
```

Comment:
add space before `=` and before/after `*` and `/` as per Hadley's and Google's style guides.